File: C++/Step4/p1546.cpp
#include <iostream>

using namespace std;
int main() {
	cin.tie(NULL);
	ios::sync_with_stdio(false);
	int n;
	cin >> n;
	double* score = new double[n];
	double max = 0.0;
	double total = 0.0;
	for (int i = 0; i < n; i++) {
        cin >> score[i];
		if (max < score[i])
			max = score[i];
	}
	for (int i = 0; i < n; i++) {
		score[i]= score[i] / max * 100;
        total += score[i];
	}

	cout << total / (double)n;
	return 0;
}

File: JAVA/Step4/p1546.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;
import java.util.StringTokenizer;

public class Main{
	public static void main(String[] args) throws IOException{
		BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter bufferedWriter = new BufferedWriter(new OutputStreamWriter(System.out));
		int n = Integer.parseInt(bufferedReader.readLine());
		String numberString = bufferedReader.readLine();
		StringTokenizer stringTokenizer = new StringTokenizer(numberString, " ");
		int scoreArray[] = new int[n];
		int max=0;
		int i=0;
		float total=0;
		while(stringTokenizer.hasMoreTokens()) {
			scoreArray[i]=Integer.parseInt(stringTokenizer.nextToken());
			if(scoreArray[i]>max) {
				max=scoreArray[i];
			}
			i++;
		}
		for (int score : scoreArray){
			total += (float)score/max*100;
		}
		bufferedWriter.write(total/n+"");
		bufferedWriter.close();
	}
}
